docs/mllib-guide.md

```diff
@@ -90,6 +90,21 @@ version 1.4 or newer.
 
 # Migration Guide
 
+## From 1.2 to 1.3
+
+In the `spark.mllib` package:
+
+* *(Breaking change)* In [`ALS`](api/scala/index.html#org.apache.spark.mllib.recommendation.ALS), the extraneous method `solveLeastSquares` has been removed.  The `DeveloperApi` method `analyzeBlocks` was also removed.
```

Comment:
Shall we try to make the sections as code tabs? It is getting longer and longer.

For the `breaking change`, we should mention that they are experimental or developer APIs. `ALS.solverLeastSquares` is perhaps the only outlier.